x-pack/plugins/security_solution/cypress/tasks/rules_bulk_edit.ts

```diff
@@ -130,6 +130,19 @@ export const waitForBulkEditActionToFinish = ({ rulesCount }: { rulesCount: numb
   cy.contains(TOASTER_BODY, `You've successfully updated ${rulesCount} rule`);
 };
 
+export const waitForBulkEditActionToFinishWithSkippedRules = ({
+  rulesCount,
+}: {
+  rulesCount: number;
+}) => {
+  cy.get(BULK_ACTIONS_PROGRESS_BTN).should('be.disabled');
+  if (rulesCount === 1) {
+    cy.contains(TOASTER_BODY, `${rulesCount} rule was skipped`);
+  } else {
+    cy.contains(TOASTER_BODY, `${rulesCount} rules were skipped`);
+  }
+};
```

Comment:
Maybe we could replace this with

```ts
export const waitForBulkEditActionToFinish = () => {
  cy.get(BULK_ACTIONS_PROGRESS_BTN).should('be.disabled');
  cy.contains(TOASTER, 'Rules updated'); // header of the toast
};

export const checkBulkEditActionToast = ({
  updatedCount,
  skippedCount,
}: {
  updatedCount?: number;
  skippedCount?: number;
}) => {
  if (updatedCount !== undefined) {
    cy.contains(TOASTER_BODY, `You've successfully updated ${updatedCount} rule`);
  }
  if (skippedCount !== undefined) {
    if (skippedCount === 1) {
      cy.contains(TOASTER_BODY, `${skippedCount} rule was skipped`);
    } else {
      cy.contains(TOASTER_BODY, `${skippedCount} rules were skipped`);
    }
  }
};
```

Or merge these two into one function.

By the way, the toast can show both failed and succeeded outcomes of a bulk action, and for every outcome can show skipped rules. We check for only one of the two outcomes here (`You've successfully updated`), maybe it's worth adding support for the failed outcome and do it explicitly.

Also, what do we show in case of a partial failure of a bulk action?

Reply:
Merged these into a unique function to which options can be into for `failed`, `updated` and `skipepd`, as well as the option to expect the additional warning for overwriting data views, as a boolean.

Reply:
Also regarding partial failures: even if only one rule fails to update, we are now displaying the toaster saying "1 rule failed to update", while the others "silently" succeed. 

Given that we have the info to know how many succeeded and how many failed, we should show that data. I will create a separate ticket to address this.

Reply:
@jpdjere have you created a ticket yet?